wandb/sdk/lib/printer.py

```diff
@@ -452,7 +453,8 @@ def display(
         *,
         level: str | int | None = None,
     ) -> None:
-        if wandb.run and wandb.run._settings.silent:
+        global_settings = wandb_setup._setup(start_service=False).settings
```

Comment:
is there a way to configure it when instantiating the printer? and default to global in case there is no run?

Reply:
I looked at that, and it's possible, but would take a bit more work. We use a printer in `_WandbInit` before we derive run settings, and I'd have to think through how to implement that neatly if we reorganize things.

Reply:
I'll give it a try though and maybe update this PR.

Reply:
Updated!